### PR TITLE
Update Onboarding Doc and Fix Tests

### PR DESCRIPTION
**Pull request by tlangs**
First PR 🥳 Updated some of the wording in the onboarding doc and updated Firestore tests to pick up the correct environment variable (the user's `GOOGLE_CLOUD_DATA_PROJECT`) to talk to the right Firestore.

### CODE REVIEW

File: docs/jade-getting-started.md [hunk 1]
```diff
@@ -298,9 +298,11 @@ export PROXY_URL=https://jade-zzz.datarepo-dev.broadinstitute.org
 export GOOGLE_CLOUD_PROJECT=broad-jade-zzz
 export GOOGLE_APPLICATION_CREDENTIALS=/tmp/jade-dev-account.json
 export GOOGLE_SA_CERT=/tmp/jade-dev-account.pem
+export GOOGLE_CLOUD_DATA_PROJECT=broad-jade-zzz-data
+export IT_JADE_API_URL=https://jade-zzz.datarepo-dev.broadinstitute.org
 ```
 
-* If you're not on a Broad computer, you may need to set the host to `localhost`
+* If you're not on a *Broad-provided* computer, you may need to set the host to `localhost`
```
Comment (fboulnois) [inline docs/jade-getting-started.md]: minor nit to match the formatting of the rest of the document:

```suggestion
* If you're not on a **Broad-provided** computer, you may need to set the host to `localhost`
```

File: docs/jade-getting-started.md [hunk 2]
```diff
@@ -313,7 +315,7 @@ export HOST=localhost
 ./gradlew bootRun         # build jade-data-repo with Spring Boot features
 ./gradlew check           # linters and unit tests
 ./gradlew testConnected   # connected tests
-./gradlew testIntegrated  # integration tests
+./gradlew testIntegration  # integration tests
```
Comment (fboulnois) [inline docs/jade-getting-started.md:318]: Nice catch 🎉

**Review by fboulnois — CHANGES_REQUESTED** [review]
Should the Firestore changes be pulled into a separate PR (or vice-versa pulling the docs into a separate PR)? Other minor comments:

**Comment by tlangs** [comment]
@fboulnois that's a good point. This was all just stuff I discovered while going through the onboarding doc. I can make the code changes be a separate PR. Pls hold.

**Review by fboulnois — APPROVED** [review]
lgtm, thank you for keeping our docs updated!

**Review by samanehsan — APPROVED** [review]
Thanks for updating this! :octocat:

**Pull request merged**